metadata.json

```diff
@@ -72,5 +77,8 @@
       "version_requirement": ">= 4.7.0 < 6.0.0"
     }
   ],
-  "description": "Module for Apache configuration"
+  "description": "Module for Apache configuration",
+  "pdk-version": "1.4.1",
+  "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
```

Comment:
It'd be nice if this was a public URL so we could have a look at it.

Reply:
The template used is showing like this because it's the cached version that was shipped with the PDK. As a rule the PDK template version used is the same as that of the PDK. The template itself can be found here: https://github.com/puppetlabs/pdk-templates